shared/router-v2/shim.native.tsx

```diff
@@ -29,7 +29,7 @@ const shimNewRoute = (Original: any, isModal: boolean, isLoggedOut: boolean) =>
     }
 
     // TODO remove and make all root views have a good background
-    wrap = <Kb.NativeView style={styles.keyboard}>{wrap}</Kb.NativeView>
+    wrap = <Kb.NativeView style={isModal ? styles.modal : styles.keyboard}>{wrap}</Kb.NativeView>
```

Comment:
we need this background since we're not drawing it due to overdraw on android now